CLI_INSTRUCTIONS.md

```diff
@@ -0,0 +1,47 @@
+# Command Line Interface Installation Instructions
+
+If you're not a fan of GUIs, but you still want to take DRM off of your
+ebooks, then this guide is for you.
+
+This guide assumes you are on Linux, but it may very well work on other
+platforms.
+
+## Step-by-step Tutorial
+
+#### Install Calibre
+   - Follow [Calibre's installation instructions](https://calibre-ebook.com/download_linux)
+
+#### Install plugins
+  - Download the DeDRM `.zip` archive from DeDRM_tools'
+     [latest release](https://github.com/apprenticeharper/DeDRM_tools/releases/latest).
+     Then unzip it.
+  - Add the DeDRM plugin to Calibre:
+     ```
+     cd *the unzipped DeDRM_tools folder*
+     calibre-customize --add DeDRM_calibre_plugin/DeDRM_plugin.zip
+     ```
+  - Add the Obok plugin:
+    ```
+    calibre-customize --add Obok_calibre_plugin/obok_plugin.zip
+    ```
+
+#### Enter your keys
+  - Figure out what format DeDRM wants your key in by looking in
+     [the code that handles that](src/prefs.py).
+     - For Kindle eInk devices, DeDRM expects you to put a list of serial
+       numbers in the `serials` field: `"serials": ["012345689abcdef"]` or
+       `"serials": ["1111111111111111", "2222222222222222"]`.
+  - Now add your keys to `$CALIBRE_CONFIG_DIRECTORY/plugins/dedrm.json`.
```

Comment:
What is the path of `$CALIBRE_CONFIG_DIRECTORY`?

Under mac, what does it stand for?

Reply:
I have solved it.